sensor/kubernetes/listener/resources/networkpolicy_test.go

```diff
@@ -400,12 +396,13 @@ func (suite *NetworkPolicyDispatcherSuite) Test_ProcessEvent() {
 				deleteMock.Times(0)
 			}
 			events := suite.dispatcher.ProcessEvent(c.netpol, c.oldNetpol, c.action)
-			reprocessDeploymentMock.Times(1)
+			deps := set.NewStringSet()
```

Comment:
nit: Maybe a `require.NotNil(events)` would be good here?

Reply:
added